public/js/pimcore/object/tags/classificationstore.js

```diff
@@ -205,22 +207,45 @@ pimcore.object.tags.classificationstore = Class.create(pimcore.object.tags.abstr
                     item.labelAlign = this.fieldConfig.labelAlign;
                 }
 
+                panelConf.listeners = {
+                    pimcoreGlobalLanguageChanged: function (language) {
+                        this.getLanguageTabListener(this.tabPanel, language, this.frontendLanguages);
+                    }.bind(this)
+                }
+
                 panelConf.items.push(item);
             }
 
-
             this.tabPanel = new Ext.TabPanel(panelConf);
 
             wrapperConfig.items = [this.tabPanel];
-
         }
 
 
         this.component = new Ext.Panel(wrapperConfig);
 
+        if (this.toolbar) {
+            this.toolbar.on(pimcore.events.globalLanguageChanged, function(language) {
+                this.tabPanel.fireEvent('pimcoreGlobalLanguageChanged', language);
+            }.bind(this));
+        }
+
         return this.component;
     },
 
+    getLanguageTabListener: function (languageTab, language, frontendLanguages) {
+        if (frontendLanguages.includes(language)) {
+            this.setActiveLanguageTab(languageTab, language)
+        }
+    },
+
+    setActiveLanguageTab: function (languageTab, language) {
+        languageTab.items.items.forEach((tabLanguage) => {
+            if (tabLanguage.currentLanguage === language) {
```

Comment:
Hi @GNi33  `tabLanguage.currentLanguage` is always `undefined`. You need to get the language of the tab from the index of the `frontendLanguages` like here: https://github.com/pimcore/admin-ui-classic-bundle/pull/292/files#diff-b1d1ac925d6b60b569f5b7a7b7edece8c7ef40174c0ff2ab8c88a21f0a440526R196. 
Could you please change that? Thanks